packages/components/Button/package.json

```diff
@@ -15,7 +15,9 @@
     "just": "fluentui-scripts",
     "clean": "fluentui-scripts clean",
     "lint": "fluentui-scripts eslint",
-    "test": "fluentui-scripts jest",
+    "test": "yarn test-iOS && yarn test-win32",
+    "test-iOS": "fluentui-scripts jest -c __tests__/iOS/jest.config.js",
+    "test-win32": "fluentui-scripts jest -c __tests__/win32/jest.config.js",
```

Comment:
I think the convention is that these are always lowercase. Also I think `test:ios` / `test:win32` is closer to what I've seen elsewhere in similar Microsoft repos